velox/exec/HashAggregation.cpp

```diff
@@ -180,12 +180,13 @@ void HashAggregation::updateRuntimeStats() {
     }
   }
 
-  runtimeStats["hashtable.capacity"] = RuntimeMetric(hashTableStats.capacity);
-  runtimeStats["hashtable.numRehashes"] =
+  runtimeStats[std::string(HashTable<true>::kCapacity)] =
```

Comment:
> HashTable<true>...

It is a bit weird to use the template parameter here. Maybe move these to HashTableBase which is not a template?

Reply:
Sure, done.